library/sbi-rt/src/base.rs

```diff
@@ -197,6 +201,10 @@ mod tests {
         assert_eq!(crate::Reset.extension_id(), 0x53525354);
         assert_eq!(crate::Pmu.extension_id(), 0x504D55);
         assert_eq!(crate::Console.extension_id(), 0x4442434E);
+        assert_eq!(crate::Sse.extension_id(), 0x535345);
+        assert_eq!(crate::Fwft.extension_id(), 0x46574654);
+        assert_eq!(crate::Dbtr.extension_id(), 0x44425452);
+        assert_eq!(crate::Mpxy.extension_id(), 0x4D505859);
```

Comment:
Assertion Order Mismatch
Please rearrange these assertions to match the order of declaration above(i.e. move the highlighted assertions to the last, after `Sta`)
```suggestion
        assert_eq!(crate::Sse.extension_id(), 0x535345);
        assert_eq!(crate::Fwft.extension_id(), 0x46574654);
        assert_eq!(crate::Dbtr.extension_id(), 0x44425452);
        assert_eq!(crate::Mpxy.extension_id(), 0x4D505859);
```